apiExamples/swapValue.html

```diff
@@ -0,0 +1,7 @@
+<auro-input id="swapExampleLeft" bordered>
```

Comment:
```suggestion
<auro-input id="swapExampleLeft" bordered value="Use the button to swap this value">
```

Reply:
Resolving without direct change due to review comments being added to a draft PR that was not ready for review.